code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2418,7 +2420,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "glass of kortara"
 	glass_desc = "The fermented nectar of the Korta nut, as enjoyed by lizards galaxywide."
 
-	
+
```

Comment:
what's with the added spaces?

Reply:
is it where I put in the extra code? Ill go through and delete it

Reply:
I've deleted the extra space